src/libs/sfdk/vmconnection.cpp

```diff
@@ -1082,6 +1082,7 @@ bool VmConnection::sshStmStep()
             BatchComposer composer = BatchComposer::createBatch("VmConnection::initGuest");
             m_guestInitBatch = composer.batch();
             emit initGuest();
+            composer.enqueueCheckPoint(this, [](){});
```

Comment:
It's a static member, so it would be better to call it as such to not suggest to the reader that it works on the `composer` instance. (It works on the currently opened batch.)

```suggestion
            BatchComposer::enqueueCheckPoint(this, [](){});
```